SU-SOE/soe_basic
# D8CORE-1579: news subtheming

# READY FOR REVIEW
- Should be merged to the 2249 since this branch is branched from there.

# Summary
- Sub theme work for the SOE News module.

# Needed By (Date)
- Tuesday

# Urgency
- Med

# Steps to Test
1. Pull in this branch into the 2249 changes
2. Pull in SOE profile branch
3. Update configs
4. Recompile
5. Check on the pages that they match the designs.

# Affected Projects or Products
- SOE Subtheming

# Affected Pages
/news -filtered and all

# Associated Issues and/or People
- D8CORE-1579
- D8CORE-2131
- D8CORE-2132
- https://github.com/SU-SOE/soe_profile/pull/50
- https://github.com/SU-SOE/soe_basic/pull/12
- Theming for the Events and News section
- @imonroe 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
